src/infi/systray/traybar.py

```diff
@@ -97,12 +122,13 @@ def shutdown(self):
         PostMessage(self._hwnd, WM_CLOSE, 0, 0)
         self._message_loop_thread.join()
 
-    def update(self, icon=None, hover_text=None):
+    def update(self, hover_text=None, icon=None):

```

Comment:
The argument position here is swapped. I prefer to leave the original order, that way it's compatible with the older version.